### PR TITLE
Send files to file list by format group

### PR DESCRIPTION
**Pull request by Hwesta**
Add format group to file info in transfers.json (script also condensed empty lists to one line), use that info to send files to file list.  Also display count of files (& selected files) in file list.

### CODE REVIEW

**Comment by Hwesta** [comment]
Can we get rid of the fixtures/file directory?  Since all the transfer & file info seems to come from transfer.json.

File: app/index.html [hunk 1]
```diff
@@ -153,6 +153,8 @@
       </div>
       <div class='panel-body' ng-controller="FileListController">
         <div ng-if="file_list.files.length > 0">
+          <p><b><ng-pluralize count="file_list.files.length" when="{'1': '1 object', 'other': '{} objects'}"></ng-pluralize></b> in list, <ng-pluralize count="selected.length" when="{'1': '1 object', 'other': '{} objects'}"></ng-pluralize> selected</p>
```
Comment (mistydemeo) [inline app/index.html]: Maybe add some newlines here for readability?

**Comment by mistydemeo** [comment]
:+1: Happy to be moving `Transfer` out of the filter, too!

**Comment by mistydemeo** [comment]
> Can we get rid of the fixtures/file directory? Since all the transfer & file info seems to come from transfer.json.

I was planning to restore it for the `Preview` pane, so that links to the pane work and render even if you have an empty file list.

**Comment by Hwesta** [comment]
> :+1: Happy to be moving Transfer out of the filter, too!

Finished removing Transfer as a dependency and fixed tests.

**Pull request merged**